test/integration/full-protocol/Loans.pairs.fork.t.sol

```diff
@@ -139,3 +201,17 @@ contract WBTCUSDC_ArbiSep_LoansForkTest is WETHUSDC_ArbiSep_LoansForkTest {
         expectedOraclePrice = 90_000_000_000;
     }
 }
+
+//// load existing
```

Comment:
this doesn't work yet, because the path to the artifacts is wrong (it's not where it expects it to be)